.github/workflows/integration_tests.yml

```diff
@@ -462,35 +420,126 @@ jobs:
           if [[ "$REMOTE_EXECUTION_PLANE" == 'true' ]]; then
             # Remote execution plane
             KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n "$KEPTN_NAMESPACE" -o jsonpath='{.data.keptn-api-token}' | base64 --decode)
-          
+            echo "remote"
             KEPTN_API_HOSTNAME=$(echo "${KEPTN_ENDPOINT}" | awk -F[/] '{print $3}')
           
-            helm install helm-service "${HELM_SERVICE_HELM_CHART_NAME}" \
-              -n ${{ env.KEPTN_NAMESPACE }}-helm-service \
-              --set remoteControlPlane.enabled=true \
-              --set remoteControlPlane.api.protocol=http \
-              --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" \
-              --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" \
-              --create-namespace
-            helm install jmeter-service "${JMETER_SERVICE_HELM_CHART_NAME}" \
-            -n ${{ env.KEPTN_NAMESPACE }}-jmeter-service \
-              --set remoteControlPlane.enabled=true \
-              --set remoteControlPlane.api.protocol=http \
-              --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" \
-              --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" \
-              --create-namespace
+            if [[ "$HELM_SERVICE_HELM_CHART_NAME" == *"dev"* ]]; then
+              echo "installing helm service from dev repo"
+              echo 'helm install helm-service ${HELM_SERVICE_HELM_CHART_NAME} --namespace ${{ env.KEPTN_NAMESPACE }}-helm-service --set remoteControlPlane.enabled=true --set remoteControlPlane.api.protocol=http --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" --create-namespace --values "./test/assets/helm-charts/helm-service/values-int-tests.yaml"'
+              helm install helm-service ${HELM_SERVICE_HELM_CHART_NAME} --namespace ${{ env.KEPTN_NAMESPACE }}-helm-service \
+                --set remoteControlPlane.enabled=true \
+                --set remoteControlPlane.api.protocol=http \
+                --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" \
+                --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" \
```

Comment:
maybe a separate remote exec plane values file would make sense to cut down on pipeline LoC?